conanfile.py

```diff
@@ -45,15 +46,18 @@ def getsubmodules():
         "fPIC": [True, False],
         "opengl": ["no", "es2", "desktop", "dynamic"],
         "openssl": ["no", "yes", "linked"],
+        "requirements": [True, False],
+        "config": "ANY"
         }, **{module: [True,False] for module in submodules}
     )
     no_copy_source = True
-    default_options = ("shared=True", "fPIC=True", "opengl=no", "openssl=no") + tuple(module + "=False" for module in submodules)
+    default_options = ("shared=False", "fPIC=True", "opengl=no", "openssl=no", "requirements=True", \
+        "config=\"\"") + tuple(module + "=False" for module in submodules)
     short_paths = True
     build_policy = "missing"
 
     def build_requirements(self):
-        if tools.os_info.is_linux:
+        if tools.os_info.is_linux and self.options.requirements:
```

Comment:
Opt out requirements install needed for core-only build, without qt gui, widgets, etc.